src/chainer.js

```diff
@@ -12,7 +13,7 @@ function chainer(functions) {
     }
 
     return result;
-  }
+  };
```

Comment:
Usage of arrow function is missing. As the task suggests, use arrow functions to make your code more concise and readable.